src/routes/spotify.ts

```diff
@@ -75,7 +76,9 @@ router.get("/authentication-callback", async (req, res) => {
     access_token: req.session.access_token,
     expires_at: req.session.expires_at
   };
-  res.redirect(`${originToRedirectTo}?${new URLSearchParams(responseData as any)}`);
+  const domain = config.client.subdirectory; //'spotify-lyrics-viewer';
+  const redirectUrl = `${originToRedirectTo}${domain}?${new URLSearchParams(responseData as any)}`;
```

Comment:
`subdirectory` would be be better term here

```suggestion
  const { subdirectory } = config.client; // For example, 'spotify-lyrics-viewer';
  const redirectUrl = `${originToRedirectTo}${subdirectory }?${new URLSearchParams(responseData as any)}`;
```